README.md

```diff
@@ -5,107 +5,104 @@
 [![kics-security-scan](https://github.com/aws-ia/terraform-aws-eks-blueprints/actions/workflows/kics-security-scan.yml/badge.svg)](https://github.com/aws-ia/terraform-aws-eks-blueprints/actions/workflows/kics-security-scan.yml)
 
-> **Note**: EKS Blueprints for Terraform is in active development and should be considered a **pre-production** framework. Backwards incompatible Terraform changes are possible in future releases and support is best-effort by the EKS Blueprints community.
+Welcome to Amazon EKS Blueprints for Terraform!
 
-Welcome to the Amazon EKS Blueprints for Terraform.
-
-This repository contains the source code for a Terraform framework that aims to accelerate the delivery of a batteries-included, multi-tenant container platform on top of Amazon EKS. This framework can be used by AWS customers, partners, and internal AWS teams to implement the foundational structure of an EKS Blueprint according to AWS best practices and recommendations.
+This repository contains a collection of Terraform modules that aim to make it easier and faster for customers to adopt [Amazon EKS](https://aws.amazon.com/eks/). It can be used by AWS customers, partners, and internal AWS teams to configure and manage complete EKS clusters that are fully bootstrapped with the operational software that is needed to deploy and operate workloads. 
 
 This project leverages the community [terraform-aws-eks](https://github.com/terraform-aws-modules/terraform-aws-eks) modules for deploying EKS Clusters.
 
 ## Getting Started
-The easiest way to get started with this framework is to follow our [Getting Started guide](./docs/getting-started.md).
+
+The easiest way to get started with EKS Blueprints is to follow our [Getting Started guide](https://aws-ia.github.io/terraform-aws-eks-blueprints/getting-started/).
 
 ## Documentation
-For complete project documentation, please visit our [documentation directory](./docs).
 
-## Patterns
-To view examples for how you can leverage this framework, see the [examples](./examples) directory.
+For complete project documentation, please visit our [documentation site](https://aws-ia.github.io/terraform-aws-eks-blueprints/).
+
+## Examples
+
+To view examples for how you can leverage EKS Blueprints, please see the [examples](./examples) directory.
 
-## Usage Example
-The below demonstrates how you can leverage this framework to deploy an EKS cluster, a managed node group, and various Kubernetes add-ons.
+## Usage
+
+The below demonstrates how you can leverage EKS Blueprints to deploy an EKS cluster, a managed node group, and various Kubernetes add-ons.
 
 ```hcl
-module "eks-blueprints" {
-    source = "github.com/aws-ia/terraform-aws-eks-blueprints"
+module "eks_blueprints" {
+  source = "github.com/aws-ia/terraform-aws-eks-blueprints"
```

Comment:
pin the version

Reply:
Now now. We can do this when we register our module with Terraform Registry. We can simply introduce a `version` variable to docs and examples